server/bleep/build.rs

```diff
@@ -21,9 +21,10 @@ fn main() {
 fn set_index_version() {
     use std::fs::{read_dir, read_to_string};
 
-    let model_directories = &["src/indexes", "src/intelligence/scope_resolution"];
+    let model_directories = &["src/intelligence/scope_resolution"];
     let model_files = &[
-        "src/indexes.rs",
+        "src/semantic/schema.rs",
```

Comment:
i believe the same holds for the semantic module as well, changes to the chunker (currently in `src/semantic/chunk.rs`) should trigger a re-index.